packages/cli/src/commands/site/deploy/index.ts

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2023 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import type { CommandModule } from "yargs";
+import type { CommonSiteArgs } from "../CommonSiteArgs";
+import type { SiteDeployArgs } from "./SiteDeployArgs";
+
+export const command: CommandModule<
+  CommonSiteArgs,
+  SiteDeployArgs
+> = {
+  command: "deploy",
+  describe: "Deploy an uploaded version",
+  builder: (argv) => {
+    return argv
+      .options({
+        "siteVersion": {
+          type: "string",
+          conflicts: "clearVersion",
+          // group: "Deploy Version",
+          // implies: { "clearVersion": "false" },
+        },
+        clearVersion: {
```

Comment:
The meaning of `clearVersion` isn't clear to me. I'm assuming it just undeploys your app entirely or something? its worth docs or a rename or something even though the underlying APIs say clearVersion.